components/eamxx/src/physics/mam/eamxx_mam_aci_process_interface.cpp

```diff
@@ -157,15 +157,16 @@ void MAMAci::set_grids(
 
   // interstitial and cloudborne aerosol tracers of interest: mass (q) and
   // number (n) mixing ratios
+  // NOTE:
+  //   - For interstitial aerosols, we have dynamics advect, but not turbulence.
+  //   - For cloudborne aerosols, DO NOT advect.
```

Comment:
We do not yet implement this though, right? That is, all tracers are AT LEAST advected by dyn.

Reply:
Ah, wait, cloudborne aerosols are not registered with `add_tracer`, right?

Reply:
Yes, the cloudborne aerosols should not be advected by HOMEE as well as SHOC. We advect them using MAM4xx advection codes.